active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
oh yeah, definitely.

On Fri, Apr 15, 2016 at 5:33 PM, Daniel McDonald <notifications@github.com>
wrote:

> In active/003-seq-feature.md
> <https://github.com/biocore/scikit-bio-rfcs/pull/9#discussion_r59958135>:
>
> > @@ -0,0 +1,108 @@
> > +---
> > +Feature name: interval-metadata
> > +Start date: <2015-12-18>
> > +Pull request: 1
> > +Authors: ["@RNAer", "@mortonjt"]
> > +Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
> > +---
> > +
> > +# Summary
> > +
> > +Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
>
> TabularMSA or other MSA objects? PSSMs?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio-rfcs/pull/9/files/a7c4b661d4f1f08e309222e0245d4394a27c8767#r59958135>
>